Assets/_BForBoss/_Weapons/Scripts/Weapons/MeleeWeapon.cs

```diff
@@ -16,6 +16,8 @@ public class MeleeWeapon
         private Action<bool> _onHitEntity;
         public bool CanMelee => _currentCooldown <= 0f;
         private int _hits;
+
+        private int layerMask = ~LayerMask.GetMask("Player");
```

Comment:
So we have this [class](https://github.com/PerigonGames/BForBoss/blob/6a287ac5b6b9127267d484e514ddc8133b678a2d/Assets/_BForBoss/_Utility/Scripts/Tags.cs#L12) that is technically meant to keeps all the layer and tags string, to reduce typing errors with these names. 

Not sure if you would rather use that?? I am down either way!

Reply:
yeah should put that in there.